src/tools/illink/test/Mono.Linker.Tests.Cases/DataFlow/FeatureGuardAttributeDataFlow.cs

```diff
@@ -19,10 +19,13 @@ namespace Mono.Linker.Tests.Cases.DataFlow
 	// Related: https://github.com/dotnet/runtime/issues/88647
 	[SetupCompileBefore ("TestFeatures.dll", new[] { "Dependencies/TestFeatures.cs" },
 		resources: new object[] { new [] { "FeatureCheckDataFlowTestSubstitutions.xml", "ILLink.Substitutions.xml" } })]
-	// FeatureGuardAttribute is currently only supported by the analyzer.
-	// The same guard behavior is achieved for ILLink/ILCompiler using substitutions.
 	[SetupCompileResource ("FeatureGuardAttributeDataFlowTestSubstitutions.xml", "ILLink.Substitutions.xml")]
 	[IgnoreSubstitutions (false)]
+	[SetupLinkerArgument ("--feature", "Mono.Linker.Tests.Cases.DataFlow.DefineFeatureGuard.FeatureSwitch", "false")]
+	[SetupLinkerArgument ("--feature", "Mono.Linker.Tests.Cases.DataFlow.DefineFeatureGuard.FeatureSwitchAndGuard", "false")]
+	[SetupLinkerArgument ("--feature", "Mono.Linker.Tests.Cases.DataFlow.FeatureGuardPrecedence.GuardAndSwitch", "true")]
+	[SetupLinkerArgument ("--feature", "Mono.Linker.Tests.Cases.DataFlow.FeatureGuardPrecedence.SwitchWithXml", "false")]
+	[SetupLinkerArgument ("--feature", "Mono.Linker.Tests.Cases.DataFlow.FeatureGuardPrecedence.GuardAndSwitchWithXml", "false")]
 	public class FeatureGuardAttributeDataFlow
```

Comment:
Should we add another test that passes `--substitutefeatureguards` and validate the feature guard getters are stubbed?